code-review-trainer-app/src/components/CodeReviewPractice.less

```diff
@@ -79,6 +79,23 @@
   border: 1px solid #ccc;
   border-radius: 4px;
   overflow: hidden;
+  text-align: left; /* Override any center alignment from parent */
```

Comment:
Please remove the comments from this commit

Reply:
I think you missed this comment. Please remove the comments you added in this commit.